js/index.js

```diff
@@ -1,7 +1,49 @@
 // Iteration 1: Names and Input
 
-
 // Iteration 2: Conditionals
 
+// Iteration 3: Loops
+
+console.log("I'm ready");
+let hacker1 = "Ciwan";
+console.log("The driver's name is " + hacker1);
+let hacker2 = "Vasiliki";
+console.log("The navigator's name is " + hacker2);
+
```

Comment:
Nice! All good on first iteration :D
As well if you guys want to try other way to put variables in a string you could try the `` quotes where you could write your string like this `The driver´s name is ${hacker1}` as an example.